### ISSUE TITLE
iOS cloud build failed

### ISSUE THREAD

**Issue by janossuta** [NativeScript/sidekick-feedback#254]
### Did you verify whether this issue has already been reported here?
Yes

### Tell us about the problem
My iOS cloud build has failed but the reason is unclear for me.

### Which platform(s) does the issue occur on?
iOS cloud build
Windows

### Provide the following version numbers that the issue occurs with:

Version: 1.12.0-v.2018.8.14.2 (latest)
NativeScript CLI version: 4.2.1
CLI extension nativescript-angular-cli version: 0.1.9
CLI extension nativescript-cloud version: 1.13.0
CLI extension nativescript-starter-kits version: 0.3.4

### Tell us how to recreate the issue in as much detail as possible
1. start the ios cloud build

### Does this issue happen every time?
Yes

### Send the client logs
https://gist.github.com/janossuta/5b929d591c0dfb8b12bf257be4327376

**Comment by ggarabedian** [comment]
@janossuta, the issue seems to be related to a plugin used in your application. Can you please delete the **node_modules** and **platforms/ios** folders, enable the **Clean Build option** in Sidekick and see if this makes any difference.
 
If the above does not help, can you please share the contents of your **package.json** file so we can attempt to reproduce the issue locally?

**Comment by janossuta** [comment]
@ggarabedian, thank you for your response I will test it when I get home but until then I can send you the package.json

```javascript{
  "nativescript": {
    "id": "org.nativescript.BorderWatcher",
    "tns-android": {
      "version": "4.2.0"
    },
    "tns-ios": {
      "version": "4.2.0"
    }
  },
  "description": "NativeScript Application",
  "license": "SEE LICENSE IN <your-license-filename>",
  "repository": "<fill-your-repository-here>",
  "dependencies": {
    "@angular/animations": "~6.1.0",
    "@angular/common": "~6.1.0",
    "@angular/compiler": "~6.1.0",
    "@angular/core": "~6.1.0",
    "@angular/forms": "~6.1.0",
    "@angular/http": "~6.1.0",
    "@angular/platform-browser": "~6.1.0",
    "@angular/platform-browser-dynamic": "~6.1.0",
    "@angular/router": "~6.1.0",
    "nativescript-angular": "~6.1.0",
    "nativescript-camera": "4.0.2",
    "nativescript-plugin-firebase": "^6.4.1",
    "nativescript-theme-core": "~1.0.4",
    "nativescript-toast": "^1.4.6",
    "nativescript-ui-chart": "^3.7.0",
    "reflect-metadata": "~0.1.10",
    "rxjs": "~6.2.0",
    "tns-core-modules": "~4.2.0",
    "zone.js": "~0.8.18"
  },
  "devDependencies": {
    "nativescript-dev-typescript": "~0.7.0",
    "nativescript-dev-webpack": "~0.15.0",
    "typescript": "~2.7.2",
    "@angular/compiler-cli": "~6.1.0",
    "@ngtools/webpack": "~6.1.0"
  },
  "readme": "NativeScript Application"
}
```

**Comment by janossuta** [comment]
@ggarabedian I tried to make a clean build after I deleted the node_modules and the iOS folder but its the same build failed. 

here is the log:
https://gist.github.com/janossuta/3eb665352c8e2abc901b5e502b145e7d

**Comment by ggarabedian** [comment]
@janossuta, thank you for sharing the package.json file. 

The issue seems to be related to the Firebase Crashlytics component and if you disable it, you should be able to build successfully. We are still not completely certain what is causing this behavior, but we assume that there is an issue with the Crashlytics implementation in the Firebase plugin. We will investigate this further and let you know if we made any progress.

In the meantime, please let me know if disabling the Crashlytics fixes the build.

**Comment by kapiltiwari61** [comment]
Hi Guys, I tried to upload ios build but it failed even i am using correct app specific password and updated sdk.
Please find logs.

`[19-04-15 20:01:26.585] Spawn failed.
[Transporter Error Output]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)
[Transporter Error Output]: ERROR ITMS-90725: "SDK Version Issue. This app was built with the iOS 11.4 SDK. All iOS apps submitted to the App Store must be built with the iOS 11 SDK or later, included in Xcode 9 or later. Further, starting March 2019, all iOS apps submitted to the App Store must be built with the iOS 12.1 SDK or later, included in Xcode 10.1 or later.", Error: Spawn failed.
[Transporter Error Output]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)
[Transporter Error Output]: ERROR ITMS-90725: "SDK Version Issue. This app was built with the iOS 11.4 SDK. All iOS apps submitted to the App Store must be built with the iOS 11 SDK or later, included in Xcode 9 or later. Further, starting March 2019, all iOS apps submitted to the App Store must be built with the iOS 12.1 SDK or later, included in Xcode 10.1 or later."
at CloudPublishService.getiOSError (/Users/jkt/.local/share/.nativescript-cli/extensions/node_modules/nativescript-cloud/lib/services/cloud-publish-service.js:79:21)
at CloudPublishService.<anonymous> (/Users/jkt/.local/share/.nativescript-cli/extensions/node_modules/nativescript-cloud/lib/services/cloud-publish-service.js:96:29)
at Generator.next (<anonymous>)
at fulfilled (/Users/jkt/.local/share/.nativescript-cli/extensions/node_modules/nativescript-cloud/lib/services/cloud-publish-service.js:4:58)
at <anonymous>
`
